benchmark_runner/common/oc/oc.py

```diff
@@ -770,7 +770,7 @@ def extract_vm_results(self, vm_name: str = '', start_stamp: str = '', end_stamp
                     continue
                 elif copy:
                     # Strip line prefix from VM output
-                    if 'cloud-init' in line:
+                    if 'cloud-init' in line and ',' in line:
```

Comment:
What are you expecting to see here?  Can you add a comment with a sample line that you expect to parse here?